validator-tests/heading-role-must-have-level.html

```diff
@@ -0,0 +1,27 @@
+<html lang="en-US">
+<head><title>Element with HEADING role MUST use the aria-level attribute to indicate the proper nesting level.</title></head>
+<body>
+
+<!--
+URL: https://www.w3.org/TR/wai-aria-1.2/#heading
+RULE: "Authors MUST use the aria-level attribute to indicate the proper nesting level."
+-->
+
+<!-- Heading element that should fail -->
+<div id="heading-1" role="heading" class="fail"></div>
+
+<div id="heading-2" role="heading" class="fail" aria-level="0"></div>
```

Comment:
While I agree that this is not valid it doesn't fail this particular rule. Can you remove this one?